web/src/cypress/integration/timepickers.ts

```diff
@@ -0,0 +1,171 @@
+import { Chance } from 'chance'
+import { DateTime } from 'luxon'
+
+import { testScreen } from '../support'
+const c = new Chance()
+
+testScreen('Time Pickers', testTimePickers)
+
+function testTimePickers(screen: ScreenFormat) {
+  describe('Time', () => {
+    const check = (name: string, params: string, display: string) =>
+      it(name, () => {
```

Comment:
Can we name these with references to where they are testing these pickers? The hardcoded commands for specific dialogs but the general titles threw me off for a sec